packages/lesswrong/components/users/DialogueMatchingPage.tsx

```diff
@@ -1059,7 +1061,9 @@ const DialogueNextStepsButton: React.FC<DialogueNextStepsButtonProps> = ({
 
   return (
     <button
-      className={classNames(classes.enterTopicsButton, isMatched ? '' : classes.hideAtXs)}
+      className={classNames(classes.enterTopicsButton, {
+        [classes.hideAtXs]: isMatched,
```

Comment:
@rjmk  This one should be `!isMatched`, presumably?

Reply:
True